### ISSUE TITLE
French Phrase

### ISSUE THREAD

**Issue by bhsmither** [cubecart/v6#1433]
In `fr-FR.xml`, `Catalogue`, `review_info`, the word 'reviews' needs translating.

**Comment by abrookbanks** [comment]
It's actually correct as it is.
![screen shot 2017-01-05 at 12 04 31](https://cloud.githubusercontent.com/assets/4520803/21680046/2298644a-d33f-11e6-9a60-b13ef41a98a3.png)

**Comment by bhsmither** [comment]
This user would say different:
https://forums.cubecart.com/topic/52006-error-on-a-translation/
For the Google translate, I think 'review' which is being used in the source French phrase in the left pane, is not being translated to English in the right pane *because it is already English*.

Other French phrases seem to be using 'avis' for the english 'review'.
```
      <string name="review" introduced="5.0.0"><![CDATA[Avis]]></string>
      <string name="review_info" introduced="5.0.0"><![CDATA[%1$s en moyenne, basé sur <a href="%3$s">%2$s reviews</a>]]></string>
      <string name="submit_review" introduced="5.0.0"><![CDATA[Soumettre l’avis]]></string>
```
Using that Google feature, with the English phrase "1 in average, based on 5 reviews" in the left pane, I get in French in the right pane:
1 en moyenne, basé sur 5 commentaires

Reversing the panes (the double arrow at top-right in the left pane), and changing 'commentaires' to 'avis', the English translation in the right pane uses 'reviews.

**Comment by briansandall** [comment]
I agree with @bhsmither - besides, in French, it should be spelled `revue`, not `review`; `avis`, which means opinion, is commonly used in translations for 'review' in the sense of providing an opinion about a product.

**Comment by abrookbanks** [comment]
So which is it? Hehe I'll ask the forums member...

**Comment by briansandall** [comment]
It should be `avis`

**Comment by abrookbanks** [comment]
Fixed with 1.0.1. Thanks guys.
